src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/Marshalling/ICustomNativeTypeMarshallingStrategy.cs

```diff
@@ -181,6 +195,16 @@ public IEnumerable<StatementSyntax> GenerateMarshalStatements(TypePositionInfo i
             {
                 yield return statement;
             }
+        }
+
+        public IEnumerable<StatementSyntax> GeneratePinnedMarshalStatements(TypePositionInfo info, StubCodeContext context)
+        {
+            var subContext = new CustomNativeTypeWithToFromNativeValueContext(context);
+
+            foreach (StatementSyntax statement in _innerMarshaller.GeneratePinnedMarshalStatements(info, subContext))
```

Comment:
I assume this is emitting things like `fixed (var p = ...)`? A comment with a general idea of what will be emitted would be helpful.

Reply:
Okay.